authors/apps/authentication/serializers.py

```diff
@@ -18,12 +18,14 @@ class RegistrationSerializer(serializers.ModelSerializer):
 
     # The client should not be able to send a token along with a registration
     # request. Making `token` read-only handles that for us.
+    token = serializers.CharField(max_length=255, read_only=True)
 
     class Meta:
         model = User
         # List all of the fields that could possibly be included in a request
         # or response, including fields specified explicitly above.
-        fields = ['email', 'username', 'password']
+        fields = ['email', 'username', 'password', 'token']
+        #extra_kwargs = {'token': {'read_only': True}}
```

Comment:
Why do we have this line in our codebase yet it is commented out?

Reply:
Am to remove it.